packages/manager/modules/billing/src/components/services-actions/services-actions.controller.js

```diff
@@ -46,7 +49,10 @@ export default class ServicesActionsCtrl {
         this.renewLink = `${this.CORE_MANAGER_URLS.telecom}sms/${this.service.serviceId}/options/recredit`;
         break;
       default:
-        this.resiliateLink = `${this.autorenewLink}/delete?serviceId=${this.service.serviceId}${serviceTypeParam}`;
+        this.resiliateLink = this.service.engagementDetails
+          ? (this.getResiliationLink && this.getResiliationLink()) ||
```

Comment:
Maybe we could set a `resiliationLink` variable to ease maintenant in reading this condition